src/components/AddingForm/AddingForm.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  FC,
+  useState,
+  useEffect,
+  Dispatch,
+  SetStateAction,
+  ChangeEvent,
+  FormEvent,
+  useRef,
+} from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { Errors } from '../../types/Errors';
+
+type Props = {
+  areAnyTodos: boolean,
+  userId: number,
+  isInputDisabled: boolean,
+  allCompleted: boolean,
+  onSubmit: (todoData: Omit<Todo, 'id'>) => Promise<void>,
+  onToggleAll: () => void,
+  setError: Dispatch<SetStateAction<Errors>>,
+};
+
+export const AddingForm: FC<Props> = ({
+  areAnyTodos,
+  userId,
+  isInputDisabled,
+  allCompleted,
+  onSubmit,
+  onToggleAll,
+  setError,
+}) => {
+  const [title, setTitle] = useState('');
+
```

Comment:
Use a more descriptive initial state, like an empty string, e.g. `useState('')`. This makes it more clear what the state value should be 🧐